lib/Exception/Reporter/Summarizer/Ix.pm

```diff
@@ -0,0 +1,69 @@
+use strict;
+use warnings;
+package Exception::Reporter::Summarizer::Ix;
+
+use parent 'Exception::Reporter::Summarizer';
+
+=head1 OVERVIEW
+
+If added as a summarizer to an L<Exception::Reporter>, this plugin will
+summarize L<Ix> exception wrappers. This should be the first
+summarizer to be checked so that it can generate a proper error message,
+otherwise your error reports may simply say C<Ix::ExceptionWrapper=HASH...>.
+
+=cut
+
+use Try::Tiny;
+
+sub new {
+  my ($class, $arg) = @_;
+  $arg ||= {};
+
+  return bless { } => $class;
+}
+
+sub can_summarize {
+  my ($self, $entry) = @_;
+  return try {
+    $entry->[1]->isa('Ix::ExceptionWrapper');
+  };
+}
+
+sub summarize {
+  my ($self, $entry) = @_;
+  my ($name, $err, $arg) = @$entry;
+
+  my @summaries;
+
+  push @summaries, $self->summarize_error($err);
+  push @summaries, $self->summarize_stack_trace($err);
+
+  return @summaries;
+}
+
+sub summarize_error {
```

Comment:
The error being handed in to the wrapper might be something that the exception report can already handle.  Like: if we have a summarizer for Error::Bananas, and the Ix wrapper exception is wrapping an Error::Bananas, then I think summarize_error should be asking the exception reporter object to figure out how to summarize, here.  So, we'd still be creating two summaries, as you do, but one of them we'd delegate back up to our E:R, which would find the right plugin.  This can only recurse if we wrap a wrapper (Weird, fine) and can only loop if we have done something *amazingly* bad.